pandas/_typing.py

```diff
@@ -209,7 +209,9 @@
 # TODO: add Ellipsis, see
 # https://github.com/python/typing/issues/684#issuecomment-548203158
 # https://bugs.python.org/issue41810
-PositionalIndexer = Union[int, np.integer, slice, Sequence[int], np.ndarray]
-PositionalIndexer2D = Union[
-    PositionalIndexer, Tuple[PositionalIndexer, PositionalIndexer]
-]
+# Using List[int] here rather than Sequence[int] to disallow tuples.
```

Comment:
can you followup and give some hints on where these should be used (a.g. a line of description for each; example....)

Reply:
> can you followup and give some hints on where these should be used (a.g. a line of description for each; example....)

Done in a new commit - added comments - let me know if you want more

Reply:
got it, looks good.